### ISSUE TITLE
Intro Video

### ISSUE THREAD

**Issue by Wanderer189** [mikesilvo164/Aeon-Nox-SiLVO#69]
Running Kodi v18-RC3 on android device with Android TV OS. Latest skin build from from here at github. 

When I enable startup intro in the skin settings, then close and try to re-open kodi, it continually crashes.

My only option is to replace the userdata>addon_data>skin.aeon.nox.silvo directory with files from a backup to get Kodi to start again.

**Comment by Wanderer189** [comment]
Just wanted to update: Same problem running Kodi v18beta2, and nightlies for beta3

**Comment by mikesilvo164** [comment]
Can you try latest from Git please?

I reworked this bit but I cannot test on Android.

**Comment by Wanderer189** [comment]
Hey Mike:
I tested it with v18 Beta 5 and here is how it turned out. With the your default intro.mp4 it worked fine. I replaced the default video with one I had used in v17.6 and it would not play, just went from splash screen, to black screen for a few seconds (12 I assume), then went on as normal.

After comparing video files I found they were both 12 seconds long, but your default video was only 2.51mb and mine was 24.9mb. They were both 1920x1080 with same audio as well.

I took mine and converted it to hvec reducing its size to 8.11mb and it then worked as well. Not sure if this is important to you, but the size thing was messing it up

Thank you Sir for getting it working. I only tested in Beta 5
